src/sentry/api/endpoints/sentry_app_authorizations.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases import SentryAppAuthorizationEndpoint as BaseEndpoint
+from sentry.api.authentication import ClientIdSecretAuthentication
+from sentry.coreapi import APIUnauthorized
+from sentry.mediators.sentry_app_installations import Authorizer
+from sentry.api.serializers.models.apitoken import ApiTokenSerializer
+
+
+class SentryAppAuthorizationsEndpoint(BaseEndpoint):
+    authentication_classes = (ClientIdSecretAuthentication, )
```

Comment:
how come this doesn't go in the `BaseEndpoint` with the permissions?

Reply:
It probably should be. Good catch.